GOVERNANCE.md

```diff
@@ -6,50 +6,27 @@ This document defines the project governance for Kubeapps.
 
 **Kubeapps**, an open-source project, is committed to building an open, inclusive, productive and self-governing open source community involved to simplifying how applications are deployed and managed in Kubernetes clusters. The community is governed by this document to define how the community should work together to achieve this goal.
```

Comment:
```suggestion
**Kubeapps**, an open-source project, is committed to building an open, inclusive, productive and self-governing open source community focused on simplifying how applications are deployed and managed in Kubernetes clusters. The community is governed by this document to define how the community should work together to achieve this goal.
```

Reply:
+1 for "focused on" rather than the other options.

Reply:
Or "involved in" (only saw that afterwards), both are fine IMO.